File: 10/java/Duck.java
/* Duck.java provides a Duck subclass of Bird.
 *
 * Begun by: Prof. Adams, for CS 214 at Calvin College.
 * Completed by: Sungmin Choi (sc94)
 * Date: March 20, 2023
 ******************************************************/

public class Duck extends Bird{
    
public Duck() {
      super();
   }

   public Duck(String name) {     
      super(name);
   }



   public String call() {
     return "Quack!";
   }

}

File: 10/ruby/Duck.rb
# Duck.rb | Defines the Duck class which inherits attributes and methods
#   from the Bird superclass.
#
# Begun by: Dr. Adams, for CS 214 at Calvin College.
# Completed by: Sungmin Choi (sc94)
# Date: March 20, 2023
####################################################

require './Bird.rb'

class Duck < Bird
    def call
        'Quack!'
    end

end